models/public/yolo-v4-tf/pre-convert.py

```diff
@@ -35,11 +37,11 @@ def main():
         '--yolo4_reorder',
     ], check=True)
 
-    subprocess.run([sys.executable, '--',
-        str(args.input_dir / 'keras-YOLOv3-model-set/tools/model_converter/keras_to_tensorflow.py'),
-        '--input_model={}'.format(args.output_dir / 'yolo-v4.h5'),
-        '--output_model={}'.format(args.output_dir / 'yolo-v4.pb'),
-    ], check=True)
+    sys.path.append(str(args.input_dir))
+    nets = importlib.import_module('keras-YOLOv3-model-set.yolo4.models.layers')
+
+    model = tf.keras.models.load_model(str(args.output_dir / 'yolo-v4.h5'), custom_objects={'mish': nets.mish})
+    model.save(filepath=args.output_dir / 'yolo-v4.savedmodel')
```

Comment:
I'm not sure this is needed. Try to just change the fourth parameter of the previous command to `yolo-v4.savedmodel`.

Reply:
Done